### ISSUE TITLE
NTR: Liver biopsy: N-acetylglutamate synthetase activity low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9215]
New term request Liver biopsy: N-acetylglutamate synthetase activity low (elthat_110106021202): NAGS low on hepatic biopsy; synthase

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000161
- label: Reduced hepatic N-acetylglutamate synthase activity
- parents: [Abnormal liver enzyme activity or concentration (HP:0034685)](https://hpo.jax.org/app/browse/term/HP:0034685)
- pmid: PMID:12594532
- def: Concentration or activity of N-acetylglutamate synthase in liver tissue below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9215
- synonyms:
